common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -162,13 +180,22 @@
             "off"
         ],
         "@typescript-eslint/no-invalid-this": [
-            "off"
+            "error"
         ],
         "@typescript-eslint/no-loss-of-precision": [
             "error"
         ],
         "@typescript-eslint/no-magic-numbers": [
-            "off"
+            "off",
```

Comment:
I see you've added configuration to no-magic-numbers, but left it off.  Is this something you intend to turn on in the future?

Reply:
I turned it on, but our assert tagging uses magic numbers, so I don't see how we can enable it without updating assert tagging, which I'd rather not tackle.